src/components/molecules/memberCard/MemberCard.module.scss

```diff
@@ -0,0 +1,57 @@
+@import "../../../foundations/sass/all";
+
+.member-card {
+  align-items: center;
+  color: $member-card-color;
+  display: flex;
+  flex-direction: column;
+  font-family: Roboto, sans-serif;
+  position: relative;
+  transition: all 300ms ease;
+  width: fit-content;
+
+  &__info {
+    align-items: center;
+    border: 3px solid $member-card-border-color;
+    border-radius: 50%; // stylelint-disable-line declaration-property-unit-allowed-list
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    gap: px-to-rem(10);
+    height: px-to-rem(343);
+    justify-content: end;
+    overflow: hidden; 
+    padding: px-to-rem(20);
+    padding-bottom: px-to-rem(70);
+    position: relative; 
+    text-align: center;
+    width: px-to-rem(343);
```

Comment:
What do you think? Could we use max-width here for responsiveness instead of width?

Reply:
It would be okay if we adjust the aspect ratio for height.

Reply:
> What do you think? Could we use max-width here for responsiveness instead of width?
sure